cmd/meroxa/root/flink/list.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright © 2022 Meroxa Inc
```

Comment:
i had this thought before, and i forgot to see if i changed it in my delete PR... should we be updating these blocks on new code to 2023? should we update old license blocks too? not a blocker on this PR

Reply:
Also updated this one!